app/views/sign_up/thank_you.html.haml

```diff
@@ -1,3 +1,4 @@
+- console
```

Comment:
we should remove this!

Reply:
whoops, yeah we should - feel free to edit anything on this branch before you merge @injms @gidsg

Reply:
✅ Done